src/test/java/org/simplify4u/plugins/ValidationChecksumTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2020 Danny van Heumen
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.simplify4u.plugins;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.io.File;
+
+import org.apache.maven.artifact.Artifact;
+import org.apache.maven.plugin.logging.Log;
+import org.simplify4u.plugins.ValidationChecksum.Builder;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+public class ValidationChecksumTest {
+
+    private static final File checksumdirectory = new File(".");
```

Comment:
please use Files.createTempDirectory in BeforeMethod - when we will want to run test in parallel this can cause conflict